MultiSigWalletOverride/migrations/2_deploy_contracts.js

```diff
@@ -0,0 +1,34 @@
+const MultisigWalletWithDailyLimit = artifacts.require('MultiSigWalletWithDailyLimit.sol')
+const MultisigWalletWithoutDailyLimit = artifacts.require('MultiSigWallet.sol')
```

Comment:
This is probably repetition of line 3 and unused

Reply:
Indeed. I left it here because I wansn't sure what kind of multisig contract we want to use. Any opinions? @boolafish ? The other option is: https://github.com/gnosis/MultiSigWallet#deploy-multisig-wallet-with-daily-limit